src/Optimus.php

```diff
@@ -115,4 +122,14 @@ public function setMode($mode)
 
         static::$mode = $mode;
     }
+
+    /**
+     * Set the maximum integer.
+     *
+     * @param int $max
+     */
+    public function setMax($max)
```

Comment:
Not sure if I would add this method. Does it make sense to change the maximum after an instance has been created?